var/spack/repos/builtin/packages/lbann/package.py

```diff
@@ -90,6 +90,11 @@ class Lbann(CMakePackage):
 
     depends_on('conduit@master +hdf5', when='+conduit')
 
+    depends_on('py-breathe', when='+docs')
+    depends_on('py-m2r', when='+docs')
```

Comment:
If these are only used to build documentation, they can probably be `type='build'`.